frontend/src/page/userAvatar.tsx

```diff
@@ -0,0 +1,24 @@
+import {API_PROFILE_AVATAR, StoredUser} from "../../../common/common";
+import Avatar from "@material-ui/core/Avatar";
+import React from "react";
+import {makeServerUrl} from "../shared/shared";
+
+interface UserAvatarProps {
+  user: StoredUser;
+}
+
+export const UserAvatar: React.FC<UserAvatarProps> = (props) => {
+  const {user} = props;
+  if (user.avatarId) {
```

Comment:
As was mentioned above with the user only having one avatar, we can get rid of `avatarId` and just use the user id here.